src/data/agile-quiz.ts

```diff
@@ -418,6 +418,16 @@ const agileQuiz = [
     Explanation:
       "In Agile, the Product Owner has the ability to cancel a Sprint.",
     Link: "https://scrumguides.org/scrum-guide.html"
+  },
+  {
+    Question: "For Agile teams, the primary measure of progress is:",
+    Answer: "Working software",
+    Distractor1: "Burndown charts",
+    Distractor2: "Kanban board",
+    Distractor3: "Customer satisfaction",
+    Explanation:
+      "Working software is the primary measure of progress in agile, according to the seventh principle of the Agile Manifesto1. This means that agile teams focus on delivering software that meets the customer’s needs and provides value. By delivering working software frequently, agile teams can demonstrate their progress and achievements",
```

Comment:
fixing of typos and capitalizing Agile

```suggestion
      "Working software is the primary measure of progress in Agile, according to the seventh principle of the Agile Manifesto. This means that agile teams focus on delivering software that meets the customer’s needs and provides value. By delivering working software frequently, Agile teams can demonstrate their progress and achievements",
```